### DISCUSSION TITLE
A simple question

### DISCUSSION THREAD

**Discussion by Gu5tavius** [qtranslate/qtranslate-xt#1240] [Q&A] [answered]
I have a site that is currently running qTranslate-X - but is starting to have some weird issues (including going into a "too many redirects" death spiral) - and I'm wondering about switching it all over to XT.

Just wondering, is there much to do for migration from X to XT, or can I just deactivate one and activate the other?

Thanks.
-G

**Comment by herrvigg** [comment, marked as answer]
Wow. I don't even know how you managed to use qTranslate-X for so long. It's outdated at every possible level, PHP, security, bugs, ... 

For the migration nothing is simpler as both still share the same options:
https://github.com/qtranslate/qtranslate-xt#im-still-using-qtranslate-x-can-i-test-qtranslate-xt

But QT-XT has evolved so I don't recommend switching back to QT-X after that. Be sure to backup your DB in any case.
There are also some specific plugins like qTranslate-slugs that have now been integrated into QT-XT. For this one a real migration is done and there's no way back (unless you restore DB or do some more advanced SQL by hand).

**Reply by Gu5tavius** [reply]
Yeah, I inherited the site as part of a redesign/rejig not that long ago. I was rather surprised to see that qTranslate-X has been orphaned since something like 2016!  Yikes.

Thank you for your thoughtful reply.  I'm going to fire it up in staging and see how it goes!
-G
